### ISSUE TITLE
Add Auto-Upgrade Feature for Seamless Chart Deployment and Version Management

### ISSUE THREAD

**Issue by aviorma** [cyclops-ui/cyclops#241]
Problem Statement:
Our team aims to streamline the deployment process by utilizing this tool to effortlessly deploy charts and seamlessly upgrade versions as soon as they are available in the Helm repository.

Proposed Solution:
Integrate a feature that enables users to auto-upgrade the chart to the latest version with the click of a button, similar to the functionality offered by flux2 (Gitops).

Alternative Solutions:

Manual version checks and upgrades can be cumbersome and prone to human error. Automating this process would significantly reduce the workload and ensure timely updates.
Implementing a notification system to alert users about new chart versions could also be beneficial, but it may not be as efficient as an automated upgrade option.

Additional Context:
By incorporating this auto-upgrade feature, users can maintain their deployments effortlessly, staying up-to-date with the latest features and fixes without manual intervention. This enhancement would greatly enhance the efficiency and usability of the tool.

**Comment by petar-cvit** [comment]
Hey @aviorma, we picked this up in the past couple of days. We added a section to the `Edit module` page that allows you to change the version of a template or load the latest version.
In the recording below, we had a Module referencing a template on version `x.x.x` (the latest version). The module was created when the latest version was `0.3.0`, and since then, we have added version `0.4.0`. In version `0.4.0` we added an additional field (`my-new-field`) to show the functionality.
You can now easily load the latest version by clicking the "lock" icon in the Template section and hitting the Load button, thus bumping from `0.3.0` to `0.4.0`.

https://github.com/cyclops-ui/cyclops/assets/72022639/71cf0ac3-7b5f-4799-bf93-e843f940cebe

You can try it yourself by installing the RC version we created by running the command below:
```bash
kubectl apply -f https://raw.githubusercontent.com/cyclops-ui/cyclops/v0.5.0-rc.1/install/cyclops-install.yaml && 
kubectl apply -f https://raw.githubusercontent.com/cyclops-ui/cyclops/v0.5.0-rc.1/install/demo-templates.yaml
```
Let me know what you think and if this would help your team

**Comment by petar-cvit** [comment]
Implemented with v0.7.0
